src/social-network-provider/twitter.com/encoding.ts

```diff
@@ -0,0 +1,63 @@
+import { batchReplace, regexMatch, regexMatchAll } from '../../utils/utils'
+import { isNil, isNull } from 'lodash-es'
+
+const ICAO9303Checksum = {
```

Comment:
What is "ICAO9303"? If it is a standard, please leave a link to the spec

Reply:
And do we need to implmenet it by ourself?

Reply:
@Jack-Works  The function was introduced in #229. I wish we had a better name for the function.

Reply:
We changed a few parameters to make cksums less likely to coincide.